internal/publish/publish.go

```diff
@@ -140,7 +140,7 @@ func (s *Server) Routes(ctx context.Context) *mux.Router {
 	r.Use(middleware.PopulateLogger(logger))
 	r.Use(middleware.ProcessMaintenance(s.config))
 
-	r.Handle("/health", server.HandleHealthz())
+	r.Handle("/health", server.HandleHealthz(s.env.Database()))
```

Comment:
I think we need at least some rate limiting here - at least on a per-instance basis
since this is not an authenticated endpoint

Reply:
Done